lib/msal-node/src/cache/Storage.ts

```diff
@@ -154,8 +154,8 @@ export class Storage extends CacheManager {
                 return (cache.appMetadata[key] as AppMetadataEntity) || null;
             }
             default: {
-                console.log('Invalid Cache Type');
-                return {};
+                throw ClientAuthError.createInvalidCacheTypeError();
+                return {}; //TODO: what about this?
```

Comment:
You can delete it. If you throw, the return statement will not be executed.

Reply:
Great, thanks!